packages/slice-machine/src/modules/availableCustomTypes/index.ts

```diff
@@ -166,13 +151,10 @@ export const availableCustomTypesReducer: Reducer<
       };
     }
 
-    case getType(renameCustomTypeCreator.success): {
+    case getType(renameAvailableCustomType): {
       const id = action.payload.renamedCustomType.id;
       const customType = state[id];
 
-      // Rename only applies for custom type with local data
```

Comment:
Don't sure to get this previous condition 😅

Reply:
Yeah.. I think it was due to typing rather than an actual case